src/main/resources/i18n/Messages.properties

```diff
@@ -211,7 +211,7 @@ duration.minutesSeconds={0,number,#}min {0,number,#}s
 duration.hourMinutes={0,number,#}h {0,number,#}min
 vsSeparator=\u0020vs\u0020
 textSeparator=,\u0020
-datePattern=dd.MM.uuuu
+datePattern={0,number,#}/{1,number,#}/{2,number,#}
```

Comment:
Not sure why this was changed, however it was meant to be a pattern that is used in `DateTimeFormatter.ofPattern` which is also the recommended way to format a date, instead of sticking it together manually.